device-types/Dell/VxRail-P670F.yaml

```diff
@@ -0,0 +1,87 @@
+---
+manufacturer: Dell
+model: VxRail P670F
+slug: dell-vxrail-p670f
+u_height: 2
+is_full_depth: true
+front_image: true
+rear_image: true
+comments: |
+  [Dell VxRail P670F](https://dl.dell.com/content/manual44265077-dell-vxrail-p670f-p670n-v670f-and-s670-installation-and-service-manual.pdf?language=en-us)
+rear-ports:
+  - name: Rear VGA
+    type: other
```

Comment:
Right now VGA is not a proper interface type to model in NetBox. It should not be added to the community devicetype-library. but you are welcome to add it to your own fork

For this PR, please remove the VGA definition